.github/workflows/codespell-private.yml

```diff
@@ -4,6 +4,40 @@
 name: codespell
 on: [push, pull_request]
 jobs:
+  test:
+    env:
+      REQUIRE_ASPELL: true
+    runs-on: ubuntu-20.04
```

Comment:
Now it is ubuntu-20.04 here and ubuntu-latest below.

Reply:
codespell.yml has ubuntu-latest as well.

Reply:
Yeah it's only the version of the aspell dictionary it actually matters for. ubuntu-latest will become ubuntu-20.04 relatively soon, then we can revert this.

Reply:
I am just a fan of consistency. ;-)